roles/zuul/defaults/main.yml

```diff
@@ -67,15 +67,14 @@ nodepool_tag: "6.0.0"
 zookeeper_tag: "3.7.0"
 zuul_tag: "6.0.0"
 
-image_list:
-  nodepool_builder: "{{ docker_registry }}/zuul/nodepool-builder:{{ nodepool_tag }}"
-  nodepool_launcher: "{{ docker_registry }}/zuul/nodepool-launcher:{{ nodepool_tag }}"
-  zuul_executor: "{{ docker_registry }}/zuul/zuul-executor:{{ zuul_tag }}"
-  zuul_web: "{{ docker_registry }}/zuul/zuul-web:{{ zuul_tag }}"
-  zuul_scheduler: "{{ docker_registry }}/zuul/zuul-scheduler:{{ zuul_tag }}"
-  zookeeper: "{{ docker_registry }}/zookeeper:{{ zookeeper_tag }}"
-  mariadb: "{{ docker_registry }}/mariadb:latest"
-  log_server: "{{ docker_registry }}/httpd:alpine"
+nodepool_builder_image: "{{ docker_registry }}/zuul/nodepool-builder:{{ nodepool_tag }}"
+nodepool_launcher_image: "{{ docker_registry }}/zuul/nodepool-launcher:{{ nodepool_tag }}"
+zuul_executor_image: "{{ docker_registry }}/zuul/zuul-executor:{{ zuul_tag }}"
+zuul_web_image: "{{ docker_registry }}/zuul/zuul-web:{{ zuul_tag }}"
+zuul_scheduler_image: "{{ docker_registry }}/zuul/zuul-scheduler:{{ zuul_tag }}"
+zookeeper_image: "{{ docker_registry }}/zookeeper:{{ zookeeper_tag }}"
+mariadb_image: "{{ docker_registry }}/mariadb:latest"
+log_server_image: "{{ docker_registry }}/httpd:alpine"
```

Comment:
For the last three names maybe we should prefix them with `zuul_` in order to make them unique to this role? Same for `zookeeper_tag` and possibly for the `nodepool*` vars?

While we don't currently deploy this role together with anything else, it could help to avoid future conflicts.

And while we're touching this, maybe also pin mariadb to a specific tag?

Reply:
Good idea. Will do this.